style.css

```
@@ -7,6 +7,19 @@
     /* Ensure the HTML `hidden` attribute always wins over any display: rule */
     [hidden] { display: none !important; }
 
+    /* Screen-reader only — visually hidden but accessible */
+    .sr-only {
+      position: absolute;
+      width: 1px;
+      height: 1px;
+      padding: 0;
+      margin: -1px;
+      overflow: hidden;
+      clip: rect(0,0,0,0);
```

Stylelint (17.4.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)